js/src/vercel.ts

```diff
@@ -183,11 +183,17 @@ function stripNonAlphanumeric(input: string) {
   return input.replace(/[-:.]/g, "");
 }
 
-function convertToDottedOrderFormat(
-  [seconds, nanoseconds]: [seconds: number, nanoseconds: number],
-  runId: string,
-  executionOrder: number
-) {
+function getDotOrder(item: {
```

Comment:
Super nit: why change the name?

Reply:
Wanted to directly pass the `RunTask` without the need and changed the name as well to make it shorter 🙂